product_docs/docs/biganimal/release/getting_started/identity_provider/aws_sso.mdx

```diff
@@ -35,28 +35,18 @@ description: "Using AWS SSO as your identity provider for your Azure account"
 
 1. Select **Save changes** at the bottom of the AWS page. 
 1. Go to your newly-created application (appearing under its display name) and select the **Attribute mappings** tab. 
-1. Add your desired attributes:
-   1. Enter BigAnimal attributes under **User attribute in the application**.
-
-      | Attributes in BigAnimal                                              |
-      | -------------------------------------------------------------------- |
-      | http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname      |
-      | http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname        |
-      | http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name           |
-      | http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier |
-      | http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress   |
-
-   2. Enter the corresponding AWS SSO attributes under **Maps to this string value or user attribute in AWS SSO**.
-
-      | Supported AWS SSO attributes |
-      | ---------------------------- |
-      | ${user:givenName}            |
-      | ${user:familyName}           |
-      | ${user:name}                 |
-      | ${user:preferredUsername}    |
-      | ${user:email}                |
-
-   3. Choose an appropriate **Format** for each attribute (for example: emailAddress).  
+1. Enter your desired attribute configuration. We reccomend the following:
+
+   | User attribute in the application | Maps to this string value or user attribute in AWS SSO | Format       |
+   | --------------------------------- | ------------------------------------------------------ | ------------ |
+   | Subject                           | ${user:email}                                          | emailAddress |
+   | <assertion_path>/givenname        | ${user:givenName}                                      | basic        |
+   | <assertion_path>/surname          | ${user:familyName}                                     | basic        |
+   | <assertion_path>/name             | ${user:preferredUsername}                              | basic        |
+   | <assertion_path>/nameidentifier   | ${user:name}                                           | basic        |
```

Comment:
I doubt if we really need to configure this for AWS SSO IdP. 

@sszhao1020 can you confirm the above matrix?

Reply:
>I doubt if we really need to configure this for AWS SSO IdP. 

I mean this line

```
 | <assertion_path>/nameidentifier   | ${user:name}                                           | basic        |
```

Reply:
Hi @xugy99 
You are right. nameidentifier is not mandatory. 
Just FYI
Only givename and surname are must. Test connection successfully with the below setting.
<img width="1199" alt="image" src="https://user-images.githubusercontent.com/89435179/172034181-61e06f08-75ad-42c0-8a09-ea7bf420befe.png">

Reply:
So we should remove the nameidentifier row from the table, correct?

Reply:
for AWS SSO, yes, we can remove it.